tests/testsuite/doc.rs

```diff
@@ -1638,3 +1639,145 @@ fn crate_versions_flag_is_overridden() {
     p.cargo("rustdoc -- --crate-version 2.0.3").run();
     asserts(output_documentation());
 }
+
+#[cargo_test]
+fn doc_fingerprint_versioning_consistent() {
+    #[derive(Debug, Serialize, Deserialize)]
+    pub struct RustDocFingerprint {
+        pub rustc_verbose_version: String,
+    }
+
+    // Test that using different Rustc versions forces a
+    // doc re-compilation producing new html, css & js files.
+
+    // Random rustc verbose version
+    let stable_rustc_verbose_version = format!(
```

Comment:
I was a bit confused by the word "stable" here.  Can this maybe be renamed to something like `old_rustc_verbose_version`?

Reply:
Sure, makes complete sense.